src/pages/event-attendees/event-attendees.ts

```diff
@@ -1,6 +1,6 @@
 import {Component} from "@angular/core";
 import {Storage} from "@ionic/storage";
-import {ToastController} from "ionic-angular";
+import { ToastController, AlertController } from "ionic-angular";
```

Comment:
It isn't absolutely necessary, but it would be nice if follow uniform style everywhere. :smiley: